spaceship/lib/spaceship/tunes/application.rb

```diff
@@ -78,9 +78,9 @@ def find(identifier, mac: false)
         #  should it be an ios or an osx app
 
         def create!(name: nil, primary_language: nil, version: nil, sku: nil, bundle_id: nil, bundle_id_suffix: nil, company_name: nil, platform: nil)
+          UI.deprecated("The `version` parameter is deprecated. Use `ensure_version!` method instead") if version
```

Comment:
A feature marked as "deprecated" should still work. But here we're just throwing out `version` entirely.

I think we should raise an exception if `version` is passed since we're ignoring it.